eventnotifier/datadog/datadog.go

```diff
@@ -53,6 +54,35 @@ func (n *Notifier) GetNotifierName() string {
 	return string(types.NotifierDriverDatadog)
 }
 
+// Notify generates a notification for generic k8s events
+func (n *Notifier) Notify(obj client.Object, event corev1.Event, notifType types.NotificationType) error {
+	dis, ok := obj.(*v1beta1.Disruption)
```

Comment:
But we end up calling this for disruptionCrons too, aren't we? in notifiersink.go, we end up passing DisruptionCrons to parseEventToNotifier which calls Notify

Reply:
Yes, we do, but to reduce the size of the PR I just add the support of `DisruptionCron` for the http notifier. This is why I added this condition.

Reply:
nitpick: why not follow the same code design as the http notified and just not do anything for the cron disruption? Would be clearer I think:

```
func (n *Notifier) Notify(obj client.Object, event corev1.Event, notifType types.NotificationType) error {
// we only support disruption notifications on Datadog for now
	switch d := obj.(type) {
	case *v1beta1.Disruption:
		return n.notifyDisruption(*d, event, notifType)
	default:
		return nil
}
```